_includes/authentication.html

```diff
@@ -8,7 +8,7 @@
         Token
     </span>
     {% when 'Oauth' %}
-    <span title="This Provider can use Oauth2 for Authentication, which means upon login the Providers Page gets opened and you login to give your DeltaChat client access to the account">
+    <span title="This Provider can use Oauth2 for Authentication, which means upon login the Providers Page gets opened and you login to give your client access to the account">
```

Comment:
```suggestion
    <span title="This Provider can use Oauth2 for Authentication, which means upon login the Providers Page gets opened and you login to give your app access to the account">
```

This is user-facing, right? users typically don't know what a "client" is I think.